posts/2021-03-19-microprofile-4-21003.adoc

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title: "A new MicroProfile milestone now available in Open Liberty 21.0.0.3"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/austin0
+author_github: https://github.com/austin0
+seo-title: A new MicroProfile milestone now available in Open Liberty 21.0.0.3 - OpenLiberty.io
+seo-description: With updates to all 8 MicroProfile components, MicroProfile 4.0 is now generally available in Open Liberty 21.0.0.3.
+blog_description: With updates to all 8 MicroProfile components, MicroProfile 4.0 is now generally available in Open Liberty 21.0.0.3.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= A new MicroProfile milestone now available in Open Liberty 21.0.0.3
+Austin Bailey <https://github.com/austin0>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+// tag::intro[]
+
+In 21.0.0.3, we have an milestone update to the MicroProfile feature with a major version increase to 4.0. This includes updates to all 8 MicroProfile components. To get it, run your app with the latest version of Open Liberty.
+
+If you're feeling curious about what's coming soon in Open Liberty, take a look at the 'beta status' features in the latest development builds, for example LDAP connection support for Kerberos authentication is now available in the Open Liberty 21.0.0.4-beta release. 
```

Comment:
are we posting the beta blog at the same time? can we add a link to that?

Reply:
Link added.